### PR TITLE
[14.0][MIG] stock_packaging_usability and stock_packaging_usability_pp

### PR DESCRIPTION
**Pull request by alexis-via**
Migration from v12 to v14.
Replaces https://github.com/OCA/stock-logistics-tracking/pull/27

### CODE REVIEW

File: stock_packaging_usability_pp/models/product_packaging.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    name = fields.Char(translate=True)
+    weight = fields.Float(
+        digits="Stock Weight",
+        string="Empty Package Weight",
+        help="Empty package weight in kg",
+    )
```
Comment (yajo) [inline stock_packaging_usability_pp/models/product_packaging.py:17]: What's the difference between this field and [this other](https://github.com/odoo/odoo/blob/1bd9f69f3f3badb7b2eb0cb81b2df9ffea71020f/addons/delivery/models/product_packaging.py#L21)?

File: stock_packaging_usability_pp/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This module modifies the behavior of the buttons *Put in new
+pack* on operations (button provided by the module
+*stock_packing_usability*) and *Put in pack* on pickings (native button):
+when you click on those buttons, Odoo will ask you to select the
+packaging (object *product.packaging*) of the package.
```
Comment (yajo) [inline stock_packaging_usability_pp/readme/USAGE.rst]: This doesn't seem like an appropriate usage section. AFAIK it should contain a step by step guide on how to exercise this module's features. Check out [the instructions in the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst).

This belongs more to a description or context. But, if I read these instructions, I don't know where to go or what to do to test the module on runboat.

File: stock_packaging_usability/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This module adds 2 buttons in the *Operations* lines of a picking:
+
+* *Put in current pack*
+
+* *Put in new pack*
+
+It is a complement of the button *Put in Pack* which is available on the picking. With these 2 new buttons, you are not obliged to set the *Done quantity* package by package.
```
Comment (yajo) [inline stock_packaging_usability/readme/USAGE.rst]: This doesn't seem like an appropriate usage section. AFAIK it should contain a step by step guide on how to exercise this module's features. Check out [the instructions in the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst).

This belongs more to a description or context. But, if I read these instructions, I don't know where to go or what to do to test the module on runboat.

**Review by yajo — CHANGES_REQUESTED** [review]

File: stock_packaging_usability_pp/models/product_packaging.py (hunk 1 above)
Comment (alexis-via) [reply stock_packaging_usability_pp/models/product_packaging.py:17]: For a pallot for example:
- weight is the weight of the pallet itself, without anything on it
- max_weight is the maximum weight the pallet can support

File: stock_packaging_usability_pp/readme/USAGE.rst (hunk 2 above)
Comment (alexis-via) [reply stock_packaging_usability_pp/readme/USAGE.rst]: I agree with your remark and I made the change

File: stock_packaging_usability/readme/USAGE.rst (hunk 3 above)
Comment (alexis-via) [reply stock_packaging_usability/readme/USAGE.rst]: I agree with your remark and I made the change

**Pull request merged**